docs/content/development/attack_mitigations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "MITRE ATT&CK Mitigations"
+date: 2021-09-30T08:18:37+03:00
+draft: true
+weight: 10
+---
+
+{{% notice info %}}
+Check out [the documentation for the MITRE ATT&CK techniques as well]({{< ref "/reports/mitre" >}}).
+{{% /notice %}}
+
+## Summary
+
```

Comment:
Add a section for general public explaining what "Attack mitigations" means. I'd do something like:
"Attack Mitigations are presented in ATT&CK report. They appear next to attack descriptions to suggest steps that can be taken to reduce the risk of that particular attack technique happening in the network. They also provide links for further reading about the mitigation on ATT&CK's official website."